tests/test_generate_envvar_docs.py

```diff
@@ -18,17 +18,92 @@
 Required
 --------
 
-* ``SECRET_KEY``: Secret key that's used for certain cryptographic utilities. Defaults to: \
-``so-secret-i-cant-believe-you-are-looking-at-this``.
+* ``SECRET_KEY``: Secret key that's used for certain cryptographic utilities. Defaults to: ``so-secret-i-cant-believe-you-are-looking-at-this``.
```

Comment:
Interesting thing was discovered thanks to this test
In the `testapp.settings` I import only Sentry settings, but all of the settings from `OAF/conf/base.py` are initialized and added to the register, which is used for documentation. 

It seems that it doesn't matter what you import from `OAF/conf/base.py` - if it was accessed, then all settings are included in the documentation.
I don't know if we should do something about it, because we expect users of the library to import all the settings

Reply:
I didn't know that either, I've created an issue to at least document this https://github.com/maykinmedia/open-api-framework/issues/108